.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+<!-- This template is a guideline; use your own judgement to write a description that is easy to read and will help get the PR reviewed quickly and accurately -->
+
+<!-- Please add a short, clear title that states what this PR changes -->
+
+## ✨ What's this?
+<!-- Describe clearly and concisely what this PR changes -->
+
+### 🔗 Relationships
+<!-- Mention any issues or PRs that are connected to this -->
+
+Closes #
+
+## 🔍 Why do we want this?
+<!-- Describe the reason for making this change -->
+
+## 🏗 How is it done?
+<!-- Describe how the changes are implemented, list the different parts the changes consist of if it's more than one -->
+
+### 💥 Breaking changes
```

Comment:
While it’s good to see breaking changes documented as part of a PR, I actually think it would be better documented in a changelog inside the repository (would also be visible in the review), since breaking changes should be documented for releases. See #173

Reply:
No harm in having this here, and who knows when the other thing gets implemented consistently.